CHANGELOG.md

```diff
@@ -13,7 +13,6 @@
 * Security: Use a `NullToken` when using the new authenticator manager in the resource access checker (#4067)
 * Mercure: Do not use data in options when deleting (#4056)
 * Doctrine: Support for foreign identifiers (#4042)
-* Doctrine: Support for binary UUID in search filter (#3774)
```

Comment:
We should probably keep this line and add a new entry saying that it has been reverted.